### PR TITLE
Add compiler version test

### PR DESCRIPTION
**Pull request by kazaamjt**
- Removed pytest.ini
- Changed the wait_for_proc_bounded timout to const.SHUTDOWN_GRACE_HARD
- Add test that checks the compiler version (not None)

### CODE REVIEW

File: tests/compiler/test_basics.py [hunk 1]
```diff
@@ -16,6 +16,14 @@
     Contact: code@inmanta.com
 """
 import inmanta.compiler as compiler
+from inmanta.util import get_compiler_version
+
+
+def test_compiler_installed():
+    """
+    If this test fails, pleade install the compiler.
```
Comment (arnaudsjs) [inline tests/compiler/test_basics.py]:
```suggestion
    If this test fails, please install the compiler.
```

File: tests/compiler/test_basics.py [hunk 2]
```diff
@@ -16,6 +16,14 @@
     Contact: code@inmanta.com
 """
 import inmanta.compiler as compiler
+from inmanta.util import get_compiler_version
+
+
+def test_compiler_installed():
+    """
+    If this test fails, please install the compiler.
```
Comment (bartv) [inline tests/compiler/test_basics.py]:
```suggestion
    If this test fails, make sure inmanta is installed. For example with pip install -e .
```
